src/css/header.css

```diff
@@ -1,23 +1,132 @@
-.header {
-  padding: 20px 0;
-  background-color: blue;
-  color: yellowgreen;
+/** ========= HEADER ========= */
+
+* {
+  box-sizing: border-box;
 }
 
-.nav-list {
+.header-container {
+  position: absolute;
+  width: 335px;
```

Comment:
rubber layout. min and max width from 320px to 375px. 335px it's (max width) for 375px screen, then calculate (the min width) for 320px.